sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -870,15 +870,24 @@ class MLPClassifier(ClassifierMixin, BaseMultilayerPerceptron):
     loss_ : float
         The current loss computed with the loss function.
 
-    coefs_ : list, length n_layers - 1
+    best_loss_ : float
+        The minimum loss reached by the solver throughout fitting.
+
+    loss_curve_ : list of shape (n_iter_,)
```

Comment:
I believe this is causing sphinx to issue a warning that failed the documentation build on the CI:

https://108647-843222-gh.circle-artifacts.com/0/doc/_changed.html

I think the following might work although I am not 100% sure:

```suggestion
    loss_curve_ : list of shape (`n_iter_`,)
```

Reply:
oh yes we already got this one before